src/runtime/random.py

```diff
@@ -31,10 +46,30 @@ def randint(lower: i32, upper: i32) -> i32:
 def _lfortran_random_int(lower: i32, upper: i32) -> i32:
     pass
 
+def uniform(a: f64, b: f64) -> f64:
+    """
+    Get a random number in the range [a, b) or [a, b] depending on rounding.
+    """
+    return a + (b - a) * random()
```

Comment:
Check that `b>=a`

Reply:
Hmm... It looks like they don't check this.
```python
>>> import random
>>> random.uniform(100., 50.)
94.69713154566529
```

Reply:
Ahh, then can leave it as it is.